### PR TITLE
Game update v1.1

### PR DESCRIPTION
**Pull request by B-2ME2**

### CODE REVIEW

**Review by Mathild0 — CHANGES_REQUESTED** [review]
in FlashcardMatchGame.cshtml and in game.css there is whitespaces

**Review by Mathild0 — CHANGES_REQUESTED** [review]
nea :D

File: WebAplicationTestMVCSOL/WebAplicationTestMVC/Controllers/FlashcardsController.cs [hunk 1]
```diff
@@ -111,5 +111,25 @@ public async Task<IActionResult> CreateFlashcard(string question, string answer,
...
+        Questions = flashcards,
+        Answers = flashcards
+    };
+
+    return View(viewModel);
+}
+
+
+
+
```
Comment (gintre) [inline WebAplicationTestMVCSOL/WebAplicationTestMVC/Controllers/FlashcardsController.cs]: why there are so many white spaces?

**Review by gintre — CHANGES_REQUESTED** [review]

**Review by Mathild0 — APPROVED** [review]

File: WebAplicationTestMVCSOL/WebAplicationTestMVC/Repositories/FlashcardRepository.cs [hunk 2]
```diff
@@ -23,5 +23,9 @@ public async Task<List<Flashcard>> GetAllBySetName(string setName)
         {
             return await _context.Flashcards.Where(f => f.SetName == setName).ToListAsync();
         }
```
Comment (gintre) [inline WebAplicationTestMVCSOL/WebAplicationTestMVC/Repositories/FlashcardRepository.cs:25]: white line required after method

File: WebAplicationTestMVCSOL/WebAplicationTestMVC/Services/FlashcardService.cs [hunk 3]
```diff
@@ -41,5 +41,13 @@ public async Task<List<Flashcard>> GetAllFlashcardsBySetName(string setName)
             List<FlashcardDTO> flashcardDTOs = flashcards.Select(f => new FlashcardDTO(f.Id, f.Question, f.Answer, f.SetName)).ToList();
             return flashcardDTOs;
         }
+       
+        public async Task<List<FlashcardDTO>> GetAllFlashcardsAsDTOs()
+        {
+            var flashcards = await _FlashCardRepository.GetAll(); 
+            return flashcards.Select(fc => new FlashcardDTO(fc.Id, fc.Question, fc.Answer, fc.SetName)).ToList();
+        }
+
```
Comment (gintre) [inline WebAplicationTestMVCSOL/WebAplicationTestMVC/Services/FlashcardService.cs:50]: white spaces?

File: WebAplicationTestMVCSOL/WebAplicationTestMVC/Views/Flashcards/FlashcardMatchGame.cshtml [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        </div>
+    </div>
+</div>
+
+<div id="noFlashcardsMessage" style="display: none;">
+    <p>No more flashcards left.</p>
+    <button id="restartOrMainPageButton">Restart or Go to Main Page</button>
+</div>
+
+
```
Comment (gintre) [inline WebAplicationTestMVCSOL/WebAplicationTestMVC/Views/Flashcards/FlashcardMatchGame.cshtml:52]: WS?

**Review by gintre — CHANGES_REQUESTED** [review]

**Pull request closed without merging**